### ISSUE TITLE
Merging sometimes gets stuck indefinitely until you click on it in the taskbar

### ISSUE THREAD

**Issue by BrentW89** [matortheeternal/zedit#266]
I had an armor/clothing merge for Skyrim VR that would merge within a few minutes on the last version and on the latest version it takes an extremely long time. I haven't timed it, but it probably takes 30minutes to an hour on my very fast 5900X CPU. I'm wondering if I should return to the previous version. 

Everything I think takes a little longer on the latest version, but it's only the armor mod that is obnoxiously long. If you search Google, you'll find other people reporting this same issue. 

EDIT: I just removed my SPID plugins from my Armor merge as someone said that's not supported (would be really nice if zMerge would warn about unsupported plugin types). In any case, the merge completed in under a minute this time. Not sure if those mods caused the issue or what, but the long merge time during unloading plugins was consistent before that.

EDIT 2: I don't know what triggers it (may have nothing to do with SPID), but when it happens, it actually just hangs forever until I click the zMerge in the task bar, then progress immediately continues. I'm certain it does this as I've now seen it happen probably 8-10 times. I compile 10 different merge files and the one that hangs can change from build to build.

**Comment by Farshnuke** [comment]
Yeah I had that bug though for me it hung when I clicked off the window of the merge process.
